mgmtsystem_survey/data/survey_survey_data.xml

```diff
@@ -0,0 +1,19 @@
+<openerp>
```

Comment:
@max3903 Yes it is, but for that it's best to use`noupdate="1"`.
On the survey, it looked like a sample to me, hence my suggestion to have it as demo data.